Services/ItemService.cs

```diff
@@ -19,6 +19,10 @@ public static GetItemsResponse GetItems()
 
         public static GetItemsResponse GetItemById(int id)
         {
+            //in this method, check if the item id is 0, if it is zero, then set response.status as failure and description as missing or null item id
+            //if the id provided in request is invalid (if such an id is not present in the list), data layer will return null, in such case set status as failure and description as invalid item id
```

Comment:
remove the comments once you are done with your changes

Reply:
ok 👍